lib/components/user/places/favorite-place-screen.js

```diff
@@ -88,8 +90,8 @@ class FavoritePlaceScreen extends Component {
 
     const clonedSchemaShape = clone(validationSchemaShape)
     clonedSchemaShape.name = yup.string()
-      .required('Please enter a name for this place')
-      .notOneOf(otherPlaceNames, 'You are already using this name for another place. Please enter a different name.')
+      .required('components.FavoritePlaceScreen.invalidName')
+      .notOneOf(otherPlaceNames, 'components.FavoritePlaceScreen.nameAlreadyUsed')
```

Comment:
If using a validation error message to determine the message ID, do not use a message ID string to avoid confusion.

Reply:
This still needs to be addressed.